util/interfaces/program-names.ts

```diff
@@ -28,7 +28,8 @@ export interface ProgramNames {
     "M1422": "M1422",
     "M1423": "M1423",
     "M1424": "M1424",
-    "M1425": "M1425"
+    "M1425": "M1425",
+    "ssp": "ssp"
```

Comment:
Could you add this to be right after `appPlus` instead? This will make sure sorting keeps it from being at the bottom of the growing list of MVPs. For ProgramNamesEnum and program-names.json as well.